Libs/MRML/Widgets/qMRMLSliceControllerWidget.cxx

```diff
@@ -2446,7 +2446,8 @@ void qMRMLSliceControllerWidget::toggleSlabReconstructionInteractive(bool intera
         }
       }
     }
-  // Now mark the nodes as modified
+  // The vtkMRMLSliceIntersectionWidget should observe slice display node modifications
+  // but as a workaround for now, trigger update by modifying all the slice nodes.
```

Comment:
Should the same workaround be added to `qMRMLSliceControllerWidget::showSlabReconstructionWidget` ?

Reply:
I assumed that the `setSlabReconstructionEnabled` call [here](https://github.com/Slicer/Slicer/blob/ddb55ffda47ef88987073ebe72d70a5353435f8b/Libs/MRML/Widgets/qMRMLSliceControllerWidget.cxx#L2413) would effectively be the same / mark that node as modified, but I may be wrong